procurement_batch_generator/wizards/procurement_batch_generator.py

```diff
@@ -55,10 +53,24 @@ def _default_lines(self):
                 })
         return res
 
+    @api.model
+    def _get_default_warehouse(self):
+        warehouses = self.env['stock.warehouse'].search(
+            [('company_id', '=', self.env.user.company_id.id)])
+        warehouse_id = warehouses and warehouses[0].id or False
+        return warehouse_id
+
     line_ids = fields.One2many(
         'procurement.batch.generator.line', 'parent_id',
-        string='Procurement Request Lines', default=_default_lines)
-
+        string='Procurement Request Lines', default=_default_lines)    
+    date = fields.Date(string="Procurement Date", 
+        default=lambda self:fields.Date.context_today(self))
+    warehouse_id = fields.Many2one(comodel_name="stock.warehouse", 
+        string="Warehouse", default=_get_default_warehouse, required=1)
+    comment = fields.Text(string="Comment")
```

Comment:
The string attribute is unecessary, will be computed from attribute name `comment` to `Comment`